opentelemetry-collector.yaml

```diff
@@ -27,7 +27,7 @@ pipeline:
     with:
       repository: https://github.com/open-telemetry/opentelemetry-collector
       tag: v${{package.version}}
-      expected-commit: a082f2e439e8f77a9a9503d54d8afea576f2d08c
+      expected-commit: bf6bd9fd26194b5aee153c9a39008b9d2e8f0af0
```

Comment:
we cannot merge this PR as long as this configuration file https://raw.githubusercontent.com/open-telemetry/opentelemetry-collector-releases/main/distributions/otelcol/manifest.yaml doesn't point to v0.105.0

Reply:
I've pushed a commit, I'm still expecting it to fail until they cut a 0.105.0 tag for the releases repo but it should be safer in the future.